code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -495,3 +497,79 @@
 
 /obj/item/crusher_trophy/wolf_ear/on_mark_detonation(mob/living/target, mob/living/user)
 	user.apply_status_effect(/datum/status_effect/speed_boost, 1 SECONDS)
+
+//cosmetic items for changing the crusher's look
+
+/obj/item/crusher_trophy/retool_kit
+	name = "crusher sword retool kit"
+	desc = "A toolkit for changing the crusher's appearance without affecting the device's function. This one will make it look like a sword."
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "retool_kit"
+	denied_type = /obj/item/crusher_trophy/retool_kit
+	var/retool_icon = "crusher_sword"
+	var/retool_inhand_icon = "crusher_sword"
+	var/retool_projectile_icon = "pulse1"
+
+/obj/item/crusher_trophy/retool_kit/effect_desc()
+	return "the crusher to have the appearance of a sword"
+
+/obj/item/crusher_trophy/retool_kit/add_to(obj/item/kinetic_crusher/PKC, mob/user)
+	. = ..()
+	if(.)
+		PKC.icon_state = retool_icon
+		PKC.current_inhand_icon_state = retool_inhand_icon
+		PKC.projectile_icon = retool_projectile_icon
+		PKC.inhand_icon_state = "[PKC.current_inhand_icon_state]0" //it's pretty safe to assume it's not being wielded
```

Comment:
```suggestion
```
i dont think ud need this line at all since ur calling update_appearance() at the end of ur proc, which should call update_icon_state() where ur already handling this